src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -849,6 +860,7 @@ class AccessorModes(Enum):
         ["Bool", "Logical", "Logical List"],
         PyParameterCommandArgumentsSubItem,
     )
+    MODELOBJECT = (["ModelObject"], PySingletonCommandArgumentsSubItem)
     GENERIC = ([], PyCommandArgumentsSubItem)
```

Comment:
@seanpearsonuk , I don't think so that this is required anymore. In that case can we remove it? If so, do we need to raise an exception in the else case @ line 873?

Reply:
@prmukherj yes we can and I'll anticipate your changes along lines we discussed. Thanks

Reply:
Yes @seanpearsonuk, those changes are already in place. Thank you